src/tests/ftest/recovery/ddb.py

```diff
@@ -83,26 +88,35 @@ def test_recovery_ddb_ls(self):
 
         errors = []
 
+        object_count = self.object_count
+        dkey_count = self.dkey_count
+        akey_count = self.akey_count
         # Insert objects with API.
-        object_count = 5
-        dkey_count = 2
-        akey_count = 1
-        insert_objects(
-            context=self.context, container=self.container, object_count=object_count,
-            dkey_count=dkey_count, akey_count=akey_count, base_dkey=self.random_dkey,
-            base_akey=self.random_akey, base_data=self.random_data)
+        if not self.standalone:
+            insert_objects(
+                context=self.context, container=self.container, object_count=object_count,
+                dkey_count=dkey_count, akey_count=akey_count, base_dkey=self.random_dkey,
+                base_akey=self.random_akey, base_data=self.random_data)
 
-        # Need to stop the server to use ddb.
-        self.get_dmg_command().system_stop()
+            # Need to stop the server to use ddb.
+            self.get_dmg_command().system_stop()
 
         # 1. Verify container UUID.
         cmd_result = ddb_command.list_component()
         # Sample output.
-        # [0] d4e0c836-17bd-4df3-b255-929732486bab
+        #   Listing contents of '/'
+        #   CONT: (/[0]) /3082b7d3-32f9-41ea-bcbf-5d6450c1b34f
         # stdout is a list which contains each line as separate element. Concatenate them
         # to single string so that we can apply regex.
         ls_out = "\n".join(cmd_result[0]["stdout"])
-        match = re.search(r"\[\d+\]\s+([a-f0-9\-]+)", ls_out)
+        # Matches the container uuid
+        uuid_regex = r"([0-f]{8}-[0-f]{4}-[0-f]{4}-[0-f]{4}-[0-f]{12})"
+        match = re.search(uuid_regex, ls_out)
+        if match is None:
+            self.log.info("##### Errors #####")
+            report_errors(test=self, errors=["Unexpected output from command, unable to parse."])
+            self.log.info("##################")
+            return  # don't continue
```

Comment:
Suggestion - `report_errors` will call `test.fail`, which ends the test immediately, so `return` isn't necessary. Also, since there is only one error, I'd recommend simplifying to something like
```suggestion
        if match is None:
            self.fail("Unexpected output from ddb command, unable to parse.")
```

Reply:
Thanks ... will update.